Framework/DataHandling/test/LoadInstrumentTest.h

```diff
@@ -140,17 +128,18 @@ class LoadInstrumentTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(output->getAxis(1)->spectraNo(257), 258);
 
     auto ids_from_map = output->getSpectrum(257).getDetectorIDs();
-    IDetector_const_sptr det_from_ws = output->getDetector(257);
+	const auto &spectrumInfo = output->spectrumInfo();
+    const auto &det_from_ws = spectrumInfo.detector(257);
```

Comment:
Is this incorrect (the same for the code above)? I thought `output->getDetector(i);` should be replaced with `const auto &det = spectrumInfo.detector(i);`